substrate/pkg/controller/substrate/infrastructure/vpc.go

```diff
@@ -52,6 +53,16 @@ func (v *VPC) Create(ctx context.Context, substrate *v1alpha1.Substrate) (reconc
 		return reconcile.Result{}, fmt.Errorf("creating VPC, %w", err)
 	}
 	substrate.Status.Infrastructure.VPCID = createVpcOutput.Vpc.VpcId
```

Comment:
Can we move setting VPCID after adding additional CIDRs? If it fails we shouldn't be setting the VPCID in status.